frontend/src/app/modules/availability/new-slot/general/general.component.sass

```diff
@@ -107,8 +107,8 @@ input[type=text]
 
 ::ng-deep .input-container
     mat-select
-        align-items: center !important
-        display: flex !important
+    align-items: center !important
+    display: flex !important
```

Comment:
Styles is defined to input-container, not to mat-select

Reply:
Fixed.